packages/flutter/lib/src/material/color_scheme.dart

```diff
@@ -8,10 +8,60 @@ import 'dart:ui' as ui;
 import 'package:flutter/foundation.dart';
 import 'package:flutter/widgets.dart';
 import 'package:material_color_utilities/material_color_utilities.dart';
+import 'package:material_color_utilities/scheme/scheme_fruit_salad.dart';
+import 'package:material_color_utilities/scheme/scheme_rainbow.dart';
 
 import 'colors.dart';
 import 'theme_data.dart';
 
+/// The algorithm used to construct a [ColorScheme] in [ColorScheme.fromSeed].
```

Comment:
Note about all of the documentation for this enum: I think most (all?) occurrences of source color should be seed color.

Reply:
I see. Thanks a lot for the suggestions! Just updated all of them:)